supertokens_python/recipe/passwordless/emaildelivery/service/smtp/pless_login.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from string import Template
+from typing import TYPE_CHECKING, Union
+
+from supertokens_python.ingredients.emaildelivery.services.smtp import \
+    GetContentResult
+from supertokens_python.supertokens import Supertokens
+
+from .pless_login_email import (html_template, otp_body, table_body,
+                                url_link_body)
+
+if TYPE_CHECKING:
+    from supertokens_python.recipe.passwordless.interfaces import \
+        TypePasswordlessEmailDeliveryInput
+
+
+def pless_email_content(input_: TypePasswordlessEmailDeliveryInput) -> GetContentResult:
+    supertokens = Supertokens.get_instance()
+    app_name = supertokens.app_info.app_name
+    code_lifetime = str(input_.code_life_time)  # TODO: Humanize
```

Comment:
TODO humanise

Reply:
@KShivendu missed this?

Reply:
Done now.

Reply:
Added tests for `humanize_time` in `test_utils.py`